resources/migrations/20200325115459-add-organizations.up.sql

```diff
@@ -0,0 +1,5 @@
+CREATE TABLE organization (
+  id VARCHAR(255) NOT NULL PRIMARY KEY,
+  modifierUserId varchar(255) NOT NULL,
+  modified timestamp NOT NULL DEFAULT CURRENT_TIMESTAMP,  data jsonb
```

Comment:
Perhaps move `data jsonb` to its own line. Also consider `NOT NULL`.

Reply:
sounds good